multiqc/modules/rna_seqc/rna_seqc.py

```diff
@@ -107,15 +108,66 @@ def rnaseqc_general_stats (self):
             'scale': 'YlGn',
             'modify': lambda x: float(x) * 100.0
         }
+        headers['Expression Profiling Efficiency'] = {
+            'title': '% Expression Efficiency',
+            'description': 'Expression Profiling Efficiency',
+            'max': 100,
+            'min': 0,
+            'suffix': '%',
+            'scale': 'RdBu',
+            'modify': lambda x: float(x) * 100.0
+        }
         headers['Genes Detected'] = {
             'title': '# Genes',
             'description': 'Number of genes detected',
             'min': 0,
             'scale': 'Bu',
             'format': '{:,.0f}'
         }
+
         self.general_stats_addcols(self.rna_seqc_metrics, headers)
 
+    def transcript_associated_plot (self):
+        """ Plot a bargraph showing the Transcripts-associated reads  """
+        # Plot bar graph of groups
+        keys = [ 'Exonic Rate', 'Intronic Rate', 'Intergenic Rate' ]
+        # Config for the plot
+        pconfig = {
+            'id': 'rna_seqc_position_plot',
+            'title': 'RNA-SeQC: Transcripts-associated reads',
+            'ylab': '% Reads',
+            'cpswitch_counts_label': '# Ratio',
+            'cpswitch_percent_label': '% Reads',
+            'ymin': 0,
+            'cpswitch_c_active': False
+        }
+        self.add_section (
+            name = 'Transcripts-associated reads',
+            anchor = 'Transcript_associated',
+            plot = bargraph.plot(self.rna_seqc_metrics, keys, pconfig)
+        )
+
+
+    def strand_barplot(self):
```

Comment:
This function appears to be an exact copy of the one below.. I think it can be removed?